recipes/fqe/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "fqe" %}
+{% set version = "0.2.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fqe-{{ version }}.tar.gz
+  sha256: 00aa2a87d516f172c4ce3407f1ed63d515241b4b9e7b2ad0ff3d8a3a77c21eda
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  host:
+    - cython
+    - numpy
+    - pip
+    - python
+    - scipy
+    - setuptools
+    - wheel
+  run:
+    - cirq-core ~=0.12.0
+    - cirq-google ~=0.12.0
+    - cython
+    - jupyter
+    - {{ pin_compatible('numpy') }}
+    - openfermion >=1.2.0
+    - pyasn1-modules ==0.2.7
+    - python
+    - scipy
+    - sphinx >=2.3.1
+
+test:
+  imports:
+    - fqe
+  commands:
+    - pip check
+  requires:
+    - pip
```

```suggestion
test:
  imports:
    - fqe
  commands:
    - pip check
    - pytest
  requires:
    - pip
    - pytest
```
This package is compiled, so don't you want to run the unit tests from `tests/` ?

---

They are not distributed through PyPI from what I can see.

---

Then please use the source releases from GitHub instead.

---

Based on https://github.com/conda-forge/staged-recipes/pull/17119 I think it's fine to keep this test but thanks for the suggestion.